router/options.go

```diff
@@ -0,0 +1,105 @@
+package router
+
+import (
+	"github.com/google/uuid"
+	"github.com/micro/go-micro/registry"
+)
+
+var (
+	// DefaultGossipAddress is default gossip bind address
+	DefaultGossipAddress = ":9093"
+	// DefaultNetworkAddress is default network bind address
+	DefaultNetworkAddress = ":9094"
+)
+
+// Options are router options
+type Options struct {
+	// ID is router id
+	ID string
+	// Address is router micro service address
+	Address string
```

Comment:
Router has no address because it's not a service, its just a library

Reply:
We've agreed to "move" the `GossipAddress` into this field i.e. `Address` is the router's "gossip address"

Reply:
Agreed